WindowsPhoneDriver/WindowsPhoneDriver.InnerDriver/Commands/GetElementAttributeCommand.cs

```diff
@@ -28,15 +31,23 @@ public override string DoImpl()
                 return this.JsonResponse(ResponseStatus.Success, null);
             }
 
-            var property = element.GetType().GetProperty(attributeName);
-            if (property == null)
+			//Support for attributes with chained names like "Control.RenderTransform.TranslateX"
+	        var finalValue = attributeName
+				.Split('.')
+				.Aggregate(
+					(object)element,
+					(currentValue, propertyName) =>
+						{
+							var prop = currentValue?.GetType().GetProperty(propertyName);
+							return currentValue == null || prop == null ? null : prop.GetValue(currentValue);
+						});
+
+            if (finalValue == null)
             {
                 return this.JsonResponse(ResponseStatus.Success, null);
             }
 
-            var attributeValue = property.GetValue(element, null).ToString();
-
-            return this.JsonResponse(ResponseStatus.Success, attributeValue);
+            return this.JsonResponse(ResponseStatus.Success, finalValue.ToString());
```

Comment:
This will allow getting basic types only (strings, int, etc., but not collections or instances of classes).
You might want to add some serialization helper if you are planning to get complex types values, see 
https://github.com/2gis/Winium.StoreApps/blob/master/Winium/Winium.StoreApps.InnerServer/Commands/GetElementAttributeCommand.cs#L93-L119 for example